pkg/cli/node.go

```diff
@@ -0,0 +1,115 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/QMSTR/qmstr/pkg/builder"
+	"github.com/QMSTR/qmstr/pkg/common"
+	"github.com/QMSTR/qmstr/pkg/gnubuilder"
+	"github.com/QMSTR/qmstr/pkg/qmstr/service"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var (
+	target, input []string
+	mode          gnubuilder.Mode
+)
+
+var nodeCmd = &cobra.Command{
+	Use:   "node",
+	Short: "Modify nodes in the database",
+	Long: `Node is used when we want to modify file nodes in the database.
+A child command has to be specified after the 'node' command, to define 
+the action to be taken in the database. For instance, to add or dump a node.`,
+	Run: func(cmd *cobra.Command, args []string) {},
```

Comment:
This should not be set if the command itself does nothing. In case someone calls "qmstrctl node" with this implementation the command will do nothing and output nothing. If it is not defined (as it should) it would notify the user about it's usage and subcommands which are the parts that do the work.